src/howitz/endpoints.py

```diff
@@ -243,15 +297,15 @@ def expand_event_row(event_id):
 
 @main.route('/events/<event_id>/collapse_row', methods=["GET"])
 def collapse_event_row(event_id):
-    event_id = int(event_id)
-    selected_events = session.get("selected_events", [])
-    expanded_events = session.get("expanded_events", [])
     try:
-        expanded_events.remove(event_id)
+        session["expanded_events"].pop(str(event_id))
```

Comment:
Prevent potential KeyError:

```suggestion
        session["expanded_events"].pop(str(event_id), None)
```